### PR TITLE
blockchain: add nodesSize and preimagesSize meter

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes

- nodesSize: Storage size of the nodes cache
- preimageSize: Storage size of the preimages cache

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by aidan-kwon — APPROVED** [review]

File: blockchain/metrics.go [hunk 1]
```diff
@@ -31,6 +31,9 @@ var (
 	cacheGetBadBlockMissMeter = metrics.NewRegisteredMeter("klay/cache/get/badblock/miss", nil)
 	cacheGetBadBlockHitMeter  = metrics.NewRegisteredMeter("klay/cache/get/badblock/hit", nil)
 
+	cacheGetNodesSizeMeter     = metrics.NewRegisteredGauge("klay/cache/get/nodessize", nil)
+	cacheGetPreimagesSizeMeter = metrics.NewRegisteredGauge("klay/cache/get/preimagessize", nil)
```
Comment (aidan-kwon) [inline blockchain/metrics.go:35]: The metric type is **Gauge**, but the names are **Meter**.

File: blockchain/metrics.go [hunk 2]
```diff
@@ -31,6 +31,9 @@ var (
 	cacheGetBadBlockMissMeter = metrics.NewRegisteredMeter("klay/cache/get/badblock/miss", nil)
 	cacheGetBadBlockHitMeter  = metrics.NewRegisteredMeter("klay/cache/get/badblock/hit", nil)
 
+	cacheGetNodesSizeMeter     = metrics.NewRegisteredGauge("klay/cache/get/nodessize", nil)
```
Comment (KimKyungup) [inline blockchain/metrics.go:34]: How about this ?
```suggestion
	trieDBNodesSizeBytesGauge     = metrics.NewRegisteredGauge("klay/triedb/nodessizebytes", nil)
```

Comment (yoomee1313) [reply blockchain/metrics.go]: Changed to suggested forms.

File: blockchain/metrics.go (hunk 1 above)
Comment (yoomee1313) [reply blockchain/metrics.go]: Thanks, I didn't notice it.. Changed it to Gauge.

**Review by KimKyungup — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Review by ehnuje — APPROVED** [review]

**Pull request merged**